static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -81,6 +81,7 @@ define([
         var win         = (opts || {}).window || window,
             viewId      = (opts || {}).viewId,
             page        = config.page,
+            ophanHelper = config.ophan,
```

Comment:
would `ophanConfig` be a better name? or can you just inline this reference?

(I spent a while trying to find the non-existent "OphanHelper" object.)